test/integration-test.py

```diff
@@ -281,31 +281,33 @@ def get_future_output(cmd, date):
     return run(cmd, env={'FAKECLOCK': date.strftime("%a %b %d %H:%M:%S UTC %Y")})
 
 def test_expired_authz_purger():
-    def expect(target_time, num):
-        expected_output = ''
-        if num is not None:
-            expected_output = 'Deleted a total of %d expired pending authorizations' % num
-
+    def expect(target_time, num, table):
         out = get_future_output("./bin/expired-authz-purger --config cmd/expired-authz-purger/config.json --yes", target_time)
-        if 'via FAKECLOCK' not in out:
```

Comment:
Why did you remove this? This is useful for catching an easy mistake when running the integration test outside of Docker.